### PR TITLE
Add Badges initiative. Update links to website-redesign and i18n.

### PR DESCRIPTION
**Pull request by amiller-gh**

### CODE REVIEW

File: STRATEGIC-INITIATIVES.md [hunk 1]
```diff
@@ -20,12 +20,13 @@ and have the support needed.
...
-| i18n               | Rachel White                    | https://github.com/nodejs/community-committee/issues/114         |
+| i18n               | Rachel White                    | https://github.com/nodejs/i18n                                   |
 | Node.js Collection | Tierney Cyren                   | https://github.com/nodejs/nodejs-collection                      |
 | NodeTogether       | Rachel White                    | https://github.com/nodejs/community-committee/issues/63          |
 | Office Hours       | Tierney Cyren                   | https://github.com/nodejs/community-committee/issues/157         |
 | User Feedback      | Michael Dawson                  | https://github.com/nodejs/user-feedback                          |
-| Website Redesign   | Olivia Hugger                   | https://github.com/nodejs/nodejs.org/issues/1534                 |
+| Website Redesign   | Olivia Hugger                   | https://github.com/nodejs/website-redesign                       |
+| Badges             | Adam Miller                     | https://github.com/nodejs/badges
```
Comment (amiller-gh) [inline STRATEGIC-INITIATIVES.md]: I a not *technically* a member yet, am I able to be listed as the Champion?

Also, see the initial PR: https://github.com/nodejs/badges/pull/1/files

**Comment by amiller-gh** [comment]
Oops, didn't see https://github.com/nodejs/community-committee/pull/243 😅

File: STRATEGIC-INITIATIVES.md (hunk 1 above)
Comment (oe) [reply STRATEGIC-INITIATIVES.md]: I think this should be fine, it's not formally defined anywhere, and a champion literally just needs to be someone who can report to the CommComm from time to time, no need to be a member.

**Comment by oe** [comment]
@amiller-gh do you wanna remove the url change stuff so it doesn't conflict anymore?

**Comment by amiller-gh** [comment]
@oe resolved 👍 Sorry about that, busy week.

**Comment by oe** [comment]
As far as I know, the badges initiative was ratified in this week's meeting, so I'm going to merge this.

**Pull request merged**